### PR TITLE
Added Turkish translations

### PR DESCRIPTION
**Pull request by emreerkan**

### CODE REVIEW

File: languages/tgmpa-tr_TR.po [hunk 1]
```diff
@@ -0,0 +1,486 @@
+msgid ""
+msgstr ""
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Project-Id-Version: TGM Plugin Activation\n"
```
Comment (GaryJones) [inline languages/tgmpa-tr_TR.po]: Can you add in a version as well, please?

**Review by GaryJones — CHANGES_REQUESTED** [review]

File: languages/tgmpa-tr_TR.po (hunk 1 above)
Comment (emreerkan) [reply languages/tgmpa-tr_TR.po]: Done

**Review by GaryJones — APPROVED** [review]

**Comment by jrfnl** [comment]
Hi @karalamalar Thank you for this translation! We love it when people make TGMPA available in a new language.
The PR unfortunately will need to be rebased (and squashed) before I can merge it. Would you like to do this yourself or shall I do it for you ?

**Comment by emreerkan** [comment]
Done. Your turn. 😄

**Comment by jrfnl** [comment]
Thank you so much! And... my part is done too ;-) (including the - as of yet local - changelog entry with your credits :heart_eyes_cat:  )

**Pull request merged**